.github/actions/rust-build-release/src/main.py

```
@@ -208,8 +212,13 @@ def _resolve_toolchain_name(
         if name in preferred:
             return name
     channel_prefix = f"{toolchain}-"
+    dotted_prefix = f"{toolchain}."
     for name in installed_names:
-        if name == toolchain or name.startswith(channel_prefix):
+        if (
+            name == toolchain
+            or name.startswith(channel_prefix)
+            or name.startswith(dotted_prefix)
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)